src/screenControl.py

```diff
@@ -100,6 +105,38 @@ def getIsSidebarMode(self):
         (maxy, maxx) = self.screenControl.getScreenDimensions()
         return maxx > 200
 
+    def outputDescriptionPane(self, lineObj):
+        if not self.getIsSidebarMode():
+            return
+        (maxy, maxx) = self.screenControl.getScreenDimensions()
+        borderX = maxx - self.WIDTH
+        startY = self.SIDEBAR_Y + 1
+        headerLine = 'Description for ' + lineObj.path + ' :'
+        descLines = [
+                        lineObj.getTimeLastAccessed(),
+                        lineObj.getTimeLastModified(),
+                        lineObj.getOwnerId(),
+                        lineObj.getSizeInBytes(),
+                        lineObj.getLengthInLines()
+                    ]
+        self.printer.addstr(startY, borderX + 2, headerLine)
+        y = startY + 2
+        for descLine in descLines:
+            self.printer.addstr(y, borderX + 2, '    * ' + descLine)
```

Comment:
not fatal, it actually wraps around to the other side like an old-school platform game haha.

added a trimming function.